core/services/chainlink/application.go

```diff
@@ -383,25 +387,29 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	}
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
-		relayers := make(map[relay.Network]relaytypes.Relayer)
+		relayers := make(map[relay.Network]func() (loop.Relayer, error))
 		if cfg.EVMEnabled() {
-			evmRelayer := evmrelay.NewRelayer(db, chains.EVM, globalLogger.Named("EVM"), cfg, keyStore)
-			relayers[relay.EVM] = evmRelayer
+			lggr := globalLogger.Named("EVM")
```

Comment:
add comment about what is going on here. i *think* we are trying to shim the existing relayer impls into the common loop relayer interface but i'm not sure and i can't tell why we want to need to do that